README.md

```diff
@@ -180,6 +180,10 @@ in various programming languages.
   by Marcin Żebrowski  
   ([GitHub](https://github.com/marcinzh/enterprise))
 
+* **eff.go**: algebraic effects for Golang  
```

Comment:
Please move this up before the **effect** entry by Michael Truog -- and while you are at it, maybe you can insert the missing trailing whitespaces in the first line of that entry. Thanks.

Reply:
done :)